script.js

```diff
@@ -21,3 +21,141 @@ cross.addEventListener('click', hideMenu);
 for (let i = 0; i < links.length; i += 1) {
   links[i].addEventListener('click', hideMenu);
 }
+
+// Projects Data to access Dynamically
+
+const projects = [
+  {
+    name: 'Multi-Post Stories Gain+Glory',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    image: '<img src="images/Snapshoot Portfolio.png" alt="Portfolio Image" />',
```

Comment:
Great job team 🤝

- [ ] [Optional] Just like the previous reviewer requested, kindly make sure the same image displays both in the popup and in the projects section.